WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/PointOfSaleItemListErrorView.swift

```diff
@@ -0,0 +1,66 @@
+import SwiftUI
+
+struct PointOfSaleItemListErrorView: View {
+    private var viewModel: any ItemListViewModelProtocol
+
+    init(viewModel: any ItemListViewModelProtocol) {
+        self.viewModel = viewModel
+    }
+
+    var errorContents: ItemListViewModel.ErrorModel {
+        guard let errorContents = viewModel.state.hasError else {
+            return ItemListViewModel.ErrorModel(title: "Unknown error",
+                                                subtitle: "Unknown error",
+                                                buttonText: "Retry")
+        }
+        return errorContents
+    }
+
+    var body: some View {
+        VStack(alignment: .center, spacing: Constants.headerSpacing) {
+            Text(Localization.productTitle)
+                .frame(maxWidth: .infinity, alignment: .leading)
+                .padding(.vertical, Constants.headerPadding)
+                .font(Constants.titleFont)
+                .foregroundColor(Color.posPrimaryTexti3)
+            Spacer()
+            POSErrorExclamationMark()
+            Text(errorContents.title)
+                .foregroundStyle(Color.posPrimaryTexti3)
+                .font(.posTitle)
+                .bold()
+            Divider()
+                .padding([.leading, .trailing], Constants.buttonPadding)
```

Comment:
Might not be necessary. I don't see divider in designs.

TfaZ4LUkEwEGrxfnEFzvJj-fi-2530_16232

Reply:
Oh, true! The screenshot I took [here](https://github.com/woocommerce/woocommerce-ios/issues/13360) does have it, it may have changed later 😅  Removed divider on 0bce21ae38028ffe5d912f2dedcaa323ac366e57